packages/www/components/Dashboard/AssetsTable/index.tsx

```diff
@@ -146,8 +146,7 @@ const AssetsTable = ({
           source: {
             children: (
               <Box>
-                {sourceUrl &&
-                sourceUrl.indexOf("https://cdn.livepeer.com") === 0
+                {sourceUrl && sourceUrl.indexOf("https://livepeercdn.com") === 0
```

Comment:
This is going to break the asset table source of import detection, should be handled differently

Reply:
perhaps we can have check for both domains here with a `NOTE` mentioning its deprecation?

Reply:
Yeah should probably check for both here.

Reply:
I was almost pushing a commit with a check to both domains in this pr, but then I thought that even that's not ok, it would be delaying it again if we hardcode another domain. If for example at some point we decide to change again or we decide to use a different delivery method, we're going to need to come back here to fix this check that not necessarily depends on the domain.  
Couldn't we detect that it's a `Live Stream` or a `Import` relying on the domain? That's even not necessarily true, as I could import a .mp4 from another asset link and that's not a `Live Stream`

Reply:
I think it's going to be handled not too much in the future as we are right now implementing the asset dashboard, I am going to just cc @FullStackTiger. For now I put the two domains and then we can do how you suggested in the very next future!